src/science/roughness/cable_roughness.F90

```diff
@@ -178,12 +173,15 @@ SUBROUTINE ruff_resist(veg, rough, ssnow, canopy, LAI_pft, HGT_pft, reducedLAIdu
 !    (or not) using [[HgtAboveSnow]] and [[LAI_eff]]
 rough%hruff =  HeightAboveSnow
 !why are we using veg% for LAI and height-although this is synced now
-IF(cable_runtime%um_radiation .OR. cable_runtime%offline ) THEN
-! LAI decreases due to snow: formerly canopy%vlaiw
-call LAI_eff( mp, veg%vlai, veg%hc, HeightAboveSnow, &
+IF(cable_runtime%um_radiation .OR. cable_runtime%esm15                         &
```

Comment:
This is the sole "meaningful" change; including ".OR. cable_runtime%esm15" to this condition. Effective LAI remains undefined otherwise on first call as it is the explicit CALL in ESM1.5 NOT the radiation call